debian/snapcraft-completion

```diff
@@ -18,6 +18,16 @@ _snapcraft()
         _filedir -d
         return 0
         ;;
+    stage | pull | build | clean | prime)
```

Comment:
1. This isn't really related to this feature. Would you mind putting it in another PR?
2. I've actually wanted this for a while, nice job! However, it'll only work on dirty trees, right? What if we implemented a `snapcraft list-parts` command and used that here instead?